app/models/question.rb

```diff
@@ -16,11 +16,11 @@ class Question < ActiveRecord::Base
     #     vote_count
     # end
 
-    def upvotes(id)
```

Comment:
Removing these params wasn't to fix voting, it was just to fix something that was a little messy.